unit_tests/test_app.py

```diff
@@ -77,3 +78,7 @@ def test_get_endpoint_not_found(self, mock_query):
 
         response = self.app.get(self.DEED_ENDPOINT + 'CD3456')
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_schema_checks(self):
+
```

Comment:
this is o.k. because it is a def within a class - hence one line only - according to flake8